TypedSyntax/src/node.jl

```diff
@@ -306,15 +307,22 @@ end
 
 function getsrc(@nospecialize(f), @nospecialize(t))
     srcrts = code_typed(f, t; debuginfo=:source, optimize=false)
-    return only(srcrts)
+    src, rt = only(srcrts)
+    if hasfield(typeof(src), :parent)
```

Comment:
`&& src.parent !== nothing`, since this field got added back

Reply:
Oh I didn't realize that, will do